articles/openstack/ostack-gs.md

```diff
@@ -64,7 +58,8 @@ You can create your network infrastructure in one of the following ways:
 
 - Using Horizon to manually create your infrastructure (see [Using Horizon to manually create your infrastructure](#using-horizon-to-manually-create-your-infrastructure))
 - Using the OpenStack Heat orchestration tool to create stacks (see [Using the OpenStack Heat orchestration tool to create stacks](#using-the-openstack-heat-orchestration-tool-to-create-stacks))
-- Using the OpenStack API and automation (see [Using the OpenStack API and automation](#using the-openstack-api-and-automation))
+- Using the OpenStack API and automation
```

Comment:
Could you put the link back in:

(see [Using the OpenStack API and automation](#using-the-openstack-api-and-automation)

(in the original text it was missing a hypen, which is why the link was broken)